packages/extension-points/README.md

```diff
@@ -0,0 +1,36 @@
+# @drupal/extension-points-bundler
+
+Tooling to bundle and transpile your JavaScript to be used with the AdminUI.
+
+## Installation
+
+Add this module to your project. All the bundling and transpile dependencies and settings are included.
+
+```
+yarn add @drupal/extension-points-bundler
+
+# Build
+./node_modules/.bin/js-drupal-build
+
+# Build for production
+./node_modules/.bin/js-drupal-build --mode=production
+```
+
+## Documentation
+
+This tooling looks for your `*.admin_ui.components.yml` file and parses it's contents to create entry points for Webpack.
+
+This tooling assumes that your source JavaScript exists at `/js/src` and it's output will be `/js/src`.
+
+The bundling process automatically externalizes `React` and `@material-ui/core`.
+
+### Widgets Configuration
+
+A single `*.admin_ui.components.yml` file can contain as many widgets as needed.
+
+```
+widgets:
+  options_buttons:
+    component: js/build/options_buttons.widget.js
+    multiple: true
```

Comment:
Could someone clarify why `multiple: true` should/shouldn't be set?

Reply:
Sure: This is about widgets that handle multiple deltas or not. For example a multi-file upload field handles multiple deltas, while a normal textfield, doesn't handle multiple ones. If the field is configured to  handle multiple deltas, https://github.com/jsdrupal/drupal-admin-ui/commit/c919fef86a0e8fa0dbd1c7f169f7d16e7c4503cb#diff-46728d046a5e7fd717af30e8f4f19283 will create multiple instances of this widget. 

This is a 1to1 copy of  the logic of drupal.

Reply:
thanks :+1: